libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -147,6 +141,11 @@ AP_BattMonitor::init()
         // call init function for each backend
         if (drivers[instance] != nullptr) {
             drivers[instance]->init();
+            // _num_instances is actually the index for looping over instances
+            // the user may have BATT_MONITOR=0 and BATT2_MONITOR=7, in which case
+            // where will be a gap, but as we always check for drivers[instances] being nullptr
```

Comment:
"..where will be.." should be "..there will be.."

Reply:
fixed